pkg/ctl/create/nodegroup.go

```diff
@@ -52,8 +52,11 @@ func createNodeGroupCmd(g *cmdutils.Grouping) *cobra.Command {
 		cmdutils.AddRegionFlag(fs, p)
 		cmdutils.AddVersionFlag(fs, cfg.Metadata, `for nodegroups "auto" and "latest" can be used to automatically inherit version from the control plane or force latest`)
 		fs.StringVarP(&clusterConfigFile, "config-file", "f", "", "load configuration from a file")
-		fs.StringVarP(&nodeGroupFilter, "only", "", "",
+
+		fs.StringSliceVar(&nodeGroupOnlyFilters, "only", nil,
 			"select a subset of nodegroups via comma-separted list of globs, e.g.: 'ng-*,nodegroup?,N*group'")
+		fs.BoolVar(&ngFilter.IgnoreAllExisting, "ignore-existing", ngFilter.IgnoreAllExisting, "ignore all existing nodegroups")
```

Comment:
I would make this the default behavior, and then print a message saying "skipped creation of existing nodegroup X". And  then possibly add a "strict" flag? Hmmm I would need to think a bit about this,

Reply:
This is actually on by default already (see [`NewNodeGroupFIlter`](https://github.com/weaveworks/eksctl/pull/653/files#diff-ed6d54018faf3cc05074f043771cea47R31)). I'm not sure what would could be a real use-case for the strict behaviour, so I'm tempted to remove the flag.

Reply:
Yes, I think I would remove the flag as well.

Reply:
👍